src/sparseml/yolov8/utils/export_samples.py

```diff
@@ -147,10 +147,20 @@ def _export_torch_outputs(
     preds = model_out
 
     # Move to cpu for exporting
-    preds = preds.detach().to("cpu")
+    # Segmentation currently supports two outputs
+    if isinstance(preds, tuple):
+        pred_0 = preds[0].detach().to("cpu")
+        pred_1 = preds[1].detach().to("cpu")
 
-    sample_output_filename = os.path.join(sample_out_dir, f"out-{file_idx}.npz")
-    numpy.savez(sample_output_filename, preds)
+        sample_output_filename = os.path.join(sample_out_dir, f"out-{file_idx}-0.npz")
+        numpy.savez(sample_output_filename, pred_0)
+
+        sample_output_filename = os.path.join(sample_out_dir, f"out-{file_idx}-1.npz")
+        numpy.savez(sample_output_filename, pred_1)
```

Comment:
let's keep both to the same file - npz supports holding multiple arrays.  you can pass them as explicit kwarg names